cnames_active.js

```diff
@@ -754,6 +754,7 @@ var cnames_active = {
   "discord-music-player": "sushibtw.github.io/discord-music-player",
   "discord-player": "discord-player-docs.netlify.app",
   "discord-portable-player": "lolollllo.github.io/discord-portable-player",
+  "discord-rankup": "x404dev.github.io/discord-rankup"
```

Comment:
```suggestion
  "discord-rankup": "x404dev.github.io/discord-rankup",
```
As the CI failure indicates, you're missing a comma at the end of the line